src/plugins/remote-cache/index.ts

```diff
@@ -33,6 +40,10 @@ async function turboRemoteCache(
     let authHeader = request.headers['authorization']
     authHeader = Array.isArray(authHeader) ? authHeader.join() : authHeader
 
+    if (request.url.includes('health')) {
```

Comment:
Instead of doing this, moving this route outside of this plugin should be better. Add a new route plugin that is unauthenticated.